src/core_plugins/input_control_vis/public/vis_controller.js

```diff
@@ -1,8 +1,14 @@
+import _ from 'lodash';
 import React from 'react';
 import { render, unmountComponentAtNode } from 'react-dom';
 import { InputControlVis } from './components/vis/vis';
 import { controlFactory } from './control/control_factory';
 
+const panelMargin = 1;
+const embeddedPanelPadding = 8;
+const inputControlVisMargin = 5;
+const flexGridSmallMargin = 4;
```

Comment:
tiny nit but I think as static const variables, these are supposed to be named like EMBEDDED_PANEL_PADDING.  Also, is there a file you grabbed these from?  Might be worthwhile to point to it so if anything looks off someone can double check the values match and know where to look.  (though who knows how long that will stay in sync for... just a suggestion, take it or leave it)